docs/database/mssql.md

```diff
@@ -6,7 +6,7 @@ linkTitle: MSSQL
 
 **Status**: [Experimental][DocumentStatus]
 
-The Semantic Conventions for the [Microsoft SQL Server](https://www.microsoft.com/sql-server) extend and override the [Database Semantic Conventions](database-spans.md)
+The Semantic Conventions for the [Microsoft SQL Server](https://www.microsoft.com/en-us/sql-server) extend and override the [Database Semantic Conventions](database-spans.md)
```

Comment:
```suggestion
The Semantic Conventions for the *Microsoft SQL Server* extend and override the [Database Semantic Conventions](database-spans.md)
```
If we can't get it to pass I think we could (at least temporarily) just drop the link as it doesn't provide additional value for this document anyway.
In the long run, we should look into options like the cache in our website repo if it continues to be unreliable otherwise.

Reply:
Let's remove it and check if it's still breaking. Because if the problem with the tool itself we should get the same error with another url